admin_ui/app/org/OrgEditRoles.js

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from '@reach/router';
+import Store from '../Store';
+import OrgEditRolesModals from './OrgEditRolesModals';
+import useDocumentTitle from '@rehooks/document-title';
+import TabLinks from '../../components/TabLinks';
+import Button from '../../components/Button';
+import Grid from '../../components/Grid';
+
+// Dummy data
+let roleHeadings = [
+  {
+    label: 'Name',
+    className: 'text-left',
+  },
+  {
+    label: 'Permissions',
+  },
+  {
+    label: 'Users',
+  },
+  {
+    label: 'Actions',
+    isSortable: false,
+    className: 'text-right',
+  },
+];
+let roleData = [
+  {
+    id: 1,
+    name: 'blog_user',
+    permissionCount: 1,
+    userCount: 10,
+  },
+  {
+    id: 2,
+    name: 'blog_admin',
+    permissionCount: 4,
+    userCount: 2,
+  },
+];
+
+const OrgEditRoles = ({ orgId }) => {
+  // Load the store (we need access to store.org.currentModal$)
+  const store = React.useContext(Store);
+
+  // Set page title
+  useDocumentTitle(`Organization name: Roles`);
+
+  return (
+    <React.Fragment>
+      <OrgEditRolesModals/>
+      <h1 className="mb-6">&quot;Organization name&quot;: Roles</h1>
+      <TabLinks
+        className="mb-6"
+        links={[
+          {
+            label: 'Org details',
+            to: `/organizations/${orgId}/edit`,
+          },
+          {
+            label: 'Permissions',
+            to: `/organizations/${orgId}/edit/permissions`,
+          },
+          {
+            label: 'Roles',
+            to: `/organizations/${orgId}/edit/roles`,
+          },
+          {
+            label: 'Users',
+            to: `/organizations/${orgId}/edit/users`,
+          },
+        ]}
+      />
+      <fieldset className="mb-6">
+        <legend>New role</legend>
+        <Button
+          onClick={() => {
+            store.org.currentRolesModal$.next('CREATE');
+          }}
+        >
+          Create new role
+        </Button>
+        <p className="mt-4">
+          Tip: If you want to create a role that is <em>not</em> scoped to the &quot;ORGNAME&quot;
+          organization, please visit the <Link to="/roles">Roles</Link> page.
+        </p>
+      </fieldset>
+      <fieldset className="mb-6">
+        <legend>Existing roles</legend>
+        <Grid
+          headings={roleHeadings}
+          data={roleData}
+          renderer={(roleData, index) => {
```

Comment:
It's a bit confusing that you use `roleData` here and `roleData` above - the former is a single element of the array, the latter is the whole array. I would suggest a different name, e.g. `roles` VS `role` or `roleArr` VS `roleItem` or `roleData` VS `e`